packages/api-providers/src/webb-context/bridge/bridge-deposit.ts

```diff
@@ -0,0 +1,17 @@
+import { DepositPayload, MixerDeposit } from '../mixer/mixer-deposit';
+import { WebbApiProvider } from '../webb-provider.interface';
+
+export abstract class BridgeDeposit<
```

Comment:
Can we consider using `AnchorDeposit`? And changing the directory to `anchor/anchor-deposit`